packages/foreman/foreman/foreman.spec

```diff
@@ -699,8 +699,10 @@ plugins required for Foreman to work.
   done
   # script content
   sed -ri 'sX/usr/bin/rakeX%{scl_rake}X' extras/dbmigrate script/foreman-rake
-
-  sed -i '/^ExecStart/ s|/usr/bin/sidekiq \(.\+\)$|/usr/bin/scl enable tfm "sidekiq \1"|' extras/systemd/%{dynflow_sidekiq_service_name}.service
+  # ruby on rails wrapper
+  sed -i '/^ExecStart/ s|/usr/share/foreman/bin/rails \(.\+\)$|/usr/libexec/%{name}/%{scl_name}-rails \1|' extras/systemd/%{name}.service
```

Comment:
The original line is
```
ExecStart=/usr/share/foreman/bin/rails server --environment $FOREMAN_ENV --port $FOREMAN_PORT --binding $FOREMAN_BIND
```
After this `sed` it becomes
```
ExecStart=/usr/libexec/%{name}/%{scl_name}-rails server --environment $FOREMAN_ENV --port $FOREMAN_PORT --binding $FOREMAN_BIND
```

So the libexec wrapper is run, and in turn runs
```
ruby server --environment $FOREMAN_ENV --port $FOREMAN_PORT --binding $FOREMAN_BIND
```
which fails with
```
May 20 11:22:04 foreman-selinux.example.com foreman[7657]: ruby: No such file or directory -- server (LoadError)
```

Either the sed should add another `rails` before the captured group or `/usr/libexec/tfm-rails` should run rails instead of just ruby

Reply:
Yeah that was a copy and paste error. Rails helper should call rails. Fixed as well as the libexec macro use.

Reply:
Why is this helper needed? Why can't the existing `bin/rails` be used with the right label? Using bash wrappers was an absolute pain when trying to keep the file descriptors. Keeping everything in pure Ruby avoided forking